### PR TITLE
Feat/add cts intergation

### PR DESCRIPTION
**Pull request by nettashviro**
Change stepper in new external share
add unfolding tab for external shares
added outgoing status page

### CODE REVIEW

File: src/components/popups/external-transfer-popup/ExternalTransferPopup.vue [hunk 1]
```diff
@@ -148,6 +191,7 @@ export default {
         info: this.info,
         classification: this.classification,
         approvers: this.approvers,
+        destination: this.externalNetworkDest, // TODO: CHANGE TO APP ID
```
Comment (Shahar-Y) [inline src/components/popups/external-transfer-popup/ExternalTransferPopup.vue:194]: Is it still on the TODO list?

File: src/components/popups/external-transfer-popup/steps/Approval.vue [hunk 2]
```diff
@@ -11,7 +12,14 @@
       </v-row>
     </div>
 
-    <div v-else-if="user.approverInfo.noUnit && !user.approverInfo.isApprover">
+    <!-- if the user has no unit and he isnt approver -->
```
Comment (Shahar-Y) [inline src/components/popups/external-transfer-popup/steps/Approval.vue:15]: Good job with the comments :+1:

File: src/i18n/en.json [hunk 3]
```diff
@@ -27,7 +32,9 @@
   "pageHeaders": {
     "SharedWithMe": "Shared With Me",
     "LastUpdated": "Last Updated",
-    "ExternalTransferd": "External Transferd",
+    "ExternalTransferDefault": "",
+    "ExternalTransferd": "External Transferd {networkName}",
+    "StatusTransferd": "Status files transferd",
```
Comment (Shahar-Y) [inline src/i18n/en.json:37]: Transferred*
Change across the code with ctrl+f

File: src/i18n/he.json [hunk 4]
```diff
@@ -123,12 +130,13 @@
     "CreatedAt": "נוצר",
     "UpdatedAt": "שונה",
     "Shared": "שותפים",
-    "ExternalShare": "שיתופים חיצוניים"
+    "ExternalShare": "שיתופים חיצוניים",
+    "ExternalShareFailed": "שיתופים חיצוניים שלא התקבלו"
```
Comment (Shahar-Y) [inline src/i18n/he.json:134]: Add in english

File: src/i18n/he.json [hunk 5]
```diff
@@ -145,20 +153,29 @@
...
-    "Header": "העברה לרשת חיצונית",
-    "Destination": "יעד",
-    "Approval": "מאשר",
+    "Header": "העברה לרשת {dest}",
+    "HeaderDestDefault": "חיצונית",
+    "HeaderDestTomcal": "תומכל",
+    "NetworkDest": "רשת יעד",
+    "NetworkDestChoose": "בחר רשת יעד",
+    "Destination": "נמענים",
+    "Approval": "מאשרים",
```
Comment (Shahar-Y) [inline src/i18n/he.json:162]: Add in english

File: src/i18n/he.json [hunk 6]
```diff
@@ -145,20 +153,29 @@
     }
   },
   "externalTransfer": {
-    "Header": "העברה לרשת חיצונית",
-    "Destination": "יעד",
-    "Approval": "מאשר",
+    "Header": "העברה לרשת {dest}",
+    "HeaderDestDefault": "חיצונית",
+    "HeaderDestTomcal": "תומכל",
```
Comment (Shahar-Y) [inline src/i18n/he.json:158]: Add in english

File: src/i18n/he.json [hunk 7]
```diff
@@ -145,20 +153,29 @@
...
+      "TOMCAL": [{ "text": "שמור" }, { "text": "סודי" }, { "text": "סודי ביותר", "enable": false }],
+      "CTS": [
+        { "text": "שמור" },
+        { "text": "סודי" },
+        { "text": "סודי ביותר - אדום", "color": "red" },
+        { "text": "סודי ביותר - סגול", "color": "purple", "enable": false },
+        { "text": "סודי ביותר - כחול", "color": "blue", "enable": false }
+      ]
+    },
+    "DisableClassification": "לא ניתן להעביר קבצים בסיווג {classificationName}",
```
Comment (Shahar-Y) [inline src/i18n/he.json:178]: Add in english

File: src/store/modules/auth.js [hunk 8]
```diff
@@ -26,9 +26,17 @@ const actions = {
     cookies.remove("kd-token");
     commit("setToken", undefined);
   },
-  async addApproverInfo({ commit }, user) {
-    const approverInfo = await usersApi.getApproverInfo(user.id);
-    commit("setApproverInfo", approverInfo);
+  async addApproverInfos({ rootState, commit }, user) {
+    var approverInfos = {};
+
+    await Promise.all(
```
Comment (Shahar-Y) [inline src/store/modules/auth.js:32]: :+1:

File: src/store/modules/configuration.js [hunk 9]
```diff
@@ -7,15 +7,31 @@ const state = {
...
+      value: "value1",
+      isDefault: true,
+    },
+    {
+      appID: "appId1",
+      approvalUiUrl: "http://approval.dropbox.rabaz.org",
+      approvalUrl: "http://approval.dropbox.rabaz.org",
+      label: "label1",
+      value: "value1",
+      isDefault: false,
```
Comment (Shahar-Y) [inline src/store/modules/configuration.js:32]: At least change the default to `appId2` :grimacing:

File: src/store/modules/files.js [hunk 10]
```diff
@@ -57,34 +52,36 @@ const actions = {
       const permissions = await filesApi.fetchSharedFiles(pageNum || 0);
       const files = permissions.files;
 
-      commit("updatePageNum", pageNum + 1)
-      commit("setFiles", files);
-      commit("setServerFilesLength", permissions.itemCount);
+      commit('updatePageNum', pageNum + 1);
+      commit('setFiles', files);
+      commit('setServerFilesLength', permissions.itemCount);
```
Comment (Shahar-Y) [inline src/store/modules/files.js:57]: WHY WOULD YOU DO THIS TO MEEEE

File: src/store/modules/files.js [hunk 11]
```diff
@@ -277,30 +265,27 @@ const actions = {
...
-        commit("deleteFile", fileID);
+        commit('deleteFile', fileID);
       });
 
-      if (failedFiles.length)
-        throw new Error("חלק מהקבצים שניסית להעביר נכשלו");
+      if (failedFiles.length) throw new Error('חלק מהקבצים שניסית להעביר נכשלו');
     } catch (err) {
-      dispatch("onError", err);
+      dispatch('onError', err);
```
Comment (Shahar-Y) [inline src/store/modules/files.js:280]: MY EYES HURT

**Review by Shahar-Y — CHANGES_REQUESTED** [review]
Address the comments, mainly small things. 
In the future, PLEASE use the linter as a separated PR. This was pure pain to watch the differences.
All in all, great job :+1:

File: src/store/modules/files.js (hunk 10 above)
Comment (nettashviro) [reply src/store/modules/files.js]: I did not notice to the prettier settings, is auto-save.
But deep down it's funny :satisfied:

File: src/store/modules/files.js (hunk 11 above)
Comment (nettashviro) [reply src/store/modules/files.js]: Then close your eyes :see_no_evil:

File: src/i18n/he.json (hunk 7 above)
Comment (nettashviro) [reply src/i18n/he.json:188]: donnneeee

File: src/components/layout/sidenav/Sidenav.vue [hunk 12]
```diff
@@ -6,83 +6,84 @@
...
+        </template>
 
-      
+        <!-- <router-link to="/statusTransferd" exact-active-class="route-active" class="sidenav-subitem">
+          <v-list-item link>
+            <v-list-item-title class="sidenav-title mr-8">{{
+              $t("sidenav.ExternalTransferOutgoingStatus")
+            }}</v-list-item-title>
+          </v-list-item>
+        </router-link> -->
```
[the remark this answers is not in the document]
Comment (nettashviro) [reply src/components/layout/sidenav/Sidenav.vue]: done, waited for this page to actually work

File: src/api/share.js [hunk 13]
```diff
@@ -29,16 +29,53 @@ export async function getPermissions(fileID) {
...
+//     console.log('eeheh');
+
+//     if (err.response.status == 404) {
+//       return [];
+//     } else {
+//       store.dispatch('onError', err);
+//     }
+//   }
+// }
+
```
[the remark this answers is not in the document]
Comment (nettashviro) [reply src/api/share.js]: done

**Comment by nettashviro** [comment]
> Address the comments, mainly small things.
> In the future, PLEASE use the linter as a separated PR. This was pure pain to watch the differences.
> All in all, great job

glad you enjoyed

File: docker-compose.yml [hunk 14]
```diff
@@ -132,13 +132,13 @@ services:
...
-      - "5601:5601"
-    depends_on:
-      - elasticsearch
+  # kibana:
+  #   image: "docker.elastic.co/kibana/kibana:7.5.0"
+  #   container_name: kibana
+  #   ports:
+  #     - "5601:5601"
+  #   depends_on:
+  #     - elasticsearch
```
[the remark this answers is not in the document]
Comment (nettashviro) [reply docker-compose.yml:141]: myyyyy computtteerrr was stuck from the kibana :(

File: src/components/popups/external-transfer-popup/ExternalTransferPopup.vue (hunk 1 above)
Comment (nettashviro) [reply src/components/popups/external-transfer-popup/ExternalTransferPopup.vue]: nope

File: src/store/modules/files.js (hunk 11 above)
Comment (Shahar-Y) [reply src/store/modules/files.js]: :cry:

**Review by Shahar-Y — CHANGES_REQUESTED** [review]
Just change the `transferd` to `transferred` with ctrl + f

done

**Comment by nettashviro** [comment]
> Just change the `transferd` to `transferred` with ctrl + f

done

**Review by Shahar-Y — APPROVED** [review]

**Pull request merged**